Revit_Core_Engine/Query/MEPConnectorsLocation.cs

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using Autodesk.Revit.DB;
+using BH.Engine.Adapters.Revit;
+using BH.oM.Adapters.Revit.Settings;
+using BH.oM.Reflection.Attributes;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Queries a Family Instance that contains MEP connectors for their locations.")]
+        [Input("mepInstance", "Revit family instance to be queried for their MEP connectors.")]
+        [Input("settings", "Revit adapter settings.")]
+        [Output("mepConnectorsLocation", "The list of points representing the connector's location points.")]
+        public static List<BH.oM.Geometry.Point> MEPConnectorsLocation(this FamilyInstance mepInstance, RevitSettings settings = null)
+        {
+            settings = settings.DefaultIfNull();
+            MEPModel mepModel = mepInstance.MEPModel;
```

Comment:
@BHoMBot is likely to complain about the lack of null check here. I would consider adding some more proofing as suggested below. Can be better though to explicitly check if `mepInstance` is null, otherwise the error will be weird.
```suggestion
            MEPModel mepModel = mepInstance?.MEPModel;
```

Reply:
I've added an explicit null check instead with a `RecordError` and such, made sense.